src/app/achievements/[slug]/page.tsx

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs/promises';
+import path from 'path';
+import { parseMarkdownToHTML } from '@saitamau-maximum/markdown-processor/server';
+import matter from 'gray-matter';
+
+interface Props {
+  params: { 
+    id: string;
+    slug: string
+  };
+  searchParams: {};
+}
+
+export async function generateStaticParams() {
+  const docsDirectory = path.join(process.cwd(), `docs`, `achievement`);
+  const docs = await fs.readdir(docsDirectory);
+  const params = [];
+  for (const doc of docs) {
+    params.push({ slug : doc.replace(/\.md$/, '') });
+  }
+  return params;
```

Comment:
[not change request]
一応こんな感じで書けるよというだけ (知っておいて損はないと思うので)

```suggestion
  return docs.map(doc => ({ slug: doc.replace(/\.md$/, '') }));
```

Reply:
ありがとうございます